src/commands/tokens/send_ft/mod.rs

```diff
@@ -57,3 +64,112 @@ impl SendFtCommand {
         )
     }
 }
+
+#[allow(clippy::too_many_arguments)]
+#[tracing::instrument(
+    name = "Creating a pre-populated transaction for signature ...",
+    skip_all
+)]
+pub fn get_prepopulated_transaction(
+    network_config: &crate::config::NetworkConfig,
+    ft_contract_account_id: &near_primitives::types::AccountId,
+    receiver_account_id: &near_primitives::types::AccountId,
+    signer_id: &near_primitives::types::AccountId,
+    amount_ft: &crate::types::ft_properties::FungibleToken,
+    memo: &Option<String>,
+    deposit: &crate::types::near_token::NearToken,
+    gas: &crate::common::NearGas,
+) -> color_eyre::eyre::Result<crate::commands::PrepopulatedTransaction> {
+    let mut transfer_args = serde_json::Map::new();
```

Comment:
I think you can just 
serde_json::json! {
 receiver_id: receiver_account_id,
 memo: memo.and(...)
 amount: ...
}

Would be easier to read

Reply:
Done